src/main/scala/com/github/mrpowers/spark/spec/ml/classification/IrisKMeansClustering.scala

```diff
@@ -0,0 +1,41 @@
+package com.github.mrpowers.spark.spec.ml
+
+import com.github.mrpowers.spark.spec.sql.SparkSessionWrapper
+import org.apache.spark.ml.feature.VectorAssembler
+import org.apache.spark.sql.DataFrame
+import org.apache.spark.ml.clustering.{KMeans, KMeansModel}
+
+object IrisKMeansClustering
```

Comment:
Can you please move this to `com.github.mrpowers.spark.spec.ml.clustering` so we can mirror the Spark package structure?